modules/admin_manual/pages/release_notes.adoc

```diff
@@ -27,6 +28,90 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.4.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.4 that need your attention.
+You can also read {owncloud-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Upgrading from ownCloud Server 10.3.x to 10.4.0 does not involve database migrations. 
+The upgrade duration is, therefore, expected to be short.
+
+=== PHP 7.0 Support Discontinued
```

Comment:
Hmm, this marketing-like capitalization wasn't done in former release notes.. Not sure whether it's necessary to change this now :)

Reply:
It doesn't have anything to do with marketing, just consistency with the rest of the documentation's style guide. For consistency in this document, I'm not going to push the point. But it'd be good to have this document inline with rest of the document.

Reply:
Yeah, ok. Just saw that the former release notes are not consistent either :) Then let's keep it and take this convention for the future.